docs/Contribute.md

```diff
@@ -0,0 +1,126 @@
+**Contribution Guidelines**
+
+Cromwell is an open-source project and we heartily welcome community contributions to both our code and our documentation. Here are some guidelines for adding documentation and recommendations on where we could use help the most. 
+
+First off, here are useful links:
+
+* Cromwell documentation: [cromwell.readthedocs.io/en/develop](http://cromwell.readthedocs.io/en/develop)
```

Comment:
ToL: Perhaps the link should be to the shorter `cromwell.readthedocs.io`, not `cromwell.readthedocs.io/en/develop`.